### DISCUSSION TITLE
How mutliwindow applications (detachable docks) tiling behavior is handled (e.g. OBS, Natron)?

### DISCUSSION THREAD

**Discussion by pjhfggij** [anametologin/krohnkite#35] [Q&A]
Why some "mutliwindow" applications can be tiled, but others cannot? I'm testing:
```
1. OBS Studio
2. Natron
3. McNeel Rhino on Wine (proprietary CAD)
```

1. I am able to detach docks from OBS Studio (e.g. "sources") and by default it detaches as a floating window that I can then tile.
2. Natron docks (e.g. "Properties") when detached can't be tiled.
3. McNeel Rhino on Wine docks and "utility windows" (e.g. Grasshopper) opens as a separate floating window that cannot be tiled.

On top of that when 2. (Natron) and 3. (Rhino) windows are out of focus their "subwindows" disappear. 

Sorry if this is more a Kwin/KDE/Wayland question rather that strictly Krohnkite.

**Comment by pjhfggij** [comment]
Not sure if an issue with rofi unable to capture keys is related to that as well

**Comment by anametologin** [comment]
I cannot help you because I have no one of this application. You can research by yourself: disable Krohnkite and see what happens, enable debug windows in Krohnkite and analyze it. For example found out which windows with properties are displayed incorrectly. To understand something I need logs, and they should be as small as possible from fresh start to problem appear. Other problem - other log.

**Reply by pjhfggij** [reply]
I will try to produce proper logs. I've tested these programs on hyprland and there their "subwindows" could be tiled (although they put themselves in floating mode by default).
